lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1073,6 +1073,12 @@ def cla(self):
             self.zaxis._set_scale(self._sharez.zaxis.get_scale())
         else:
             self.zaxis._set_scale('linear')
+            try:
+                self.set_zlim(0, 1)
+            except TypeError:
+                pass
+            except AttributeError:
```

Comment:
The `AttributeError` is ignored because calling `set_zlim` activates a callback which handles other stuff when the zlim changes (not familiar with it). I am guessing that when the axes are first created this callback doesn't exist so it is safe to do? Another option is `emit=False` but then this always ignores the callback.

Does @efiring knows about this code? I can dig deeper when I have more time but honestly I am not very familiar with this code.